urllib3/util/retry.py

```diff
@@ -205,7 +206,7 @@ def get_backoff_time(self):
         return min(self.BACKOFF_MAX, backoff_value)
 
     def parse_retry_after(self, retry_after):
-        if retry_after.isdigit():
+        if re.match(r"^\s*[0-9]+\s*$", retry_after):
```

Comment:
Can we get a comment here that references the relevant RFC to explain why we allow whitespace?